### PR TITLE
add theme to import/export activity

### PR DESCRIPTION
**Pull request by calvinhu4**
Re: Issue #1976

### CODE REVIEW

**Comment by calvinhu4** [comment]
fixed that attribute issue

File: res/drawable/import_export_item_background.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <shape  android:shape="rectangle">
```
Comment (mcginty) [inline res/drawable/import_export_item_background.xml]: just one space

File: res/drawable/import_export_item_background_dark.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <shape  android:shape="rectangle">
```
Comment (mcginty) [inline res/drawable/import_export_item_background_dark.xml]: space

File: res/drawable/import_export_item_background_light.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <shape  android:shape="rectangle">
```
Comment (mcginty) [inline res/drawable/import_export_item_background_light.xml]: space

**Comment by mcginty** [comment]
merged as e64c0676362451f1861ea8b2439fc191831fa702

**Pull request closed without merging**